eisbuk-admin/src/components/atoms/DateSwitcher/__tests__/DateSwitcher.test.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import "@testing-library/jest-dom";
+import { screen, render } from "@testing-library/react";
+import React from "react";
+import * as reactRedux from "react-redux";
+
+import DateSwitcher from "../DateSwitcher";
+import AdapterDateFns from "@mui/lab/AdapterLuxon";
+import LocalizationProvider from "@mui/lab/LocalizationProvider";
+
+import { testDateLuxon } from "@/__testData__/date";
+import { changeCalendarDate } from "@/store/actions/appActions";
+import { DateTime } from "luxon";
+
+const mockDispatch = jest.fn();
+jest.spyOn(reactRedux, "useDispatch").mockImplementation(() => mockDispatch);
+const mockSelector = jest.spyOn(reactRedux, "useSelector");
+mockSelector.mockImplementation(() => testDateLuxon);
+
+describe("Date Switcher", () => {
+  describe("Smoke Test", () => {
+    const testDateISO = "2021-08-12";
+    const testDateDateTime = DateTime.fromISO(testDateISO);
+    test("should render component", () => {
+      render(
+        <LocalizationProvider dateAdapter={AdapterDateFns}>
+          <DateSwitcher
+            currentDate={testDateDateTime}
+            open={true}
+            anchorEl={null}
+            handleClose={() => {}}
+          />
+        </LocalizationProvider>
+      );
+    });
```

Comment:
No need for smoke test, after you've written the stuff down below...successful rendering is implied for all of the other tests